README.md

```diff
@@ -1,2 +1,80 @@
-# mujtaba_charm
+# mujtaba-charm
 This is a python package.
+
+
+- [Project Description](#project-description)
+- [Installation](#installation)
+- [Usage Section](#usage-section)
+- [Testing](#testing)
+
+## Project Description
+The project aims to provide a python package using poetry which has 3 sub-packages:
+1. analysis
+2. models
+3. utils
+
+## Installation
+1. Install pytest:
+```bash
+poetry add --dev pytest
+```
+
+2. Ensure all dependencies, including pytest, are installed.
+```bash
+poetry install
+```
```

Comment:
this is project setup, not package installation. write the command to install `mujtaba_charm`

Reply:
Fixed